qcsrc/server.qc

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Can you rename this to better denote that it's QW-specific?

Reply:
It's called server.qc in the original progs, so I figured I would leave it the same. It's kind of a bad name though since it really has nothing to do with server functions. It's just a file for redefining functions that were left out of qw's codebase.